MyCefet.Api/MyCefet.Api.csproj

```diff
@@ -35,4 +35,10 @@
     <Folder Include="Interfaces\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="MyCefet.Api.xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
```

Comment:
XML documentation was disable on the project properties because the xml file's path was set to your particular repository on your machine.